homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
This doesn't solve the case where sessions expire *while* HASS is running and both sensors try to login at the same time.